Osiris/Hacks/Misc.cpp

```diff
@@ -66,6 +66,51 @@ void Misc::slowwalk(UserCmd* cmd) noexcept
     }
 }
 
+void Misc::playerBlocker(UserCmd* cmd) noexcept
+{
+    if ( config.misc.playerBlocker && GetAsyncKeyState(config.misc.playerBlockerKey)) {
+        auto localPlayer = interfaces.entityList->getEntity(interfaces.engine->getLocalPlayer());
+        float bestdist = 250.f;
+        int index = -1;
+        for (int i = 1; i <= interfaces.engine->getMaxClients(); i++) {
+            auto entity = interfaces.entityList->getEntity(i);
+
+            if (!entity)
+                continue;
+
+            if (!entity->isAlive() || entity->isDormant() || entity == localPlayer)
+                continue;
+
+            float dist = distance(localPlayer->origin(), entity->origin());
+            if (dist < bestdist)
+            {
+                bestdist = dist;
+                index = i;
+            }
+        }
+
+        if (index == -1)
+            return;
+
+        auto target = interfaces.entityList->getEntity(index);
+
+        if (!target)
+            return;
+
+        Vector angles = Misc::calculateRelativeAngle(localPlayer->origin(), target->origin());
+
+        angles.y -= localPlayer->eyeAngles().y;
+        angles.normalize();
+        angles.y = std::clamp(angles.y, -180.f, 180.f);
+
+        if (angles.y < 0.0f)
```

Comment:
make a treshhold to remove jiggeling ex `if (angles.y < 1.0f)` and the other one `if (angles.y > 1.0f)`, needs to be tuned tho